Demo-SwiftPM/WherebySDKDemo.xcodeproj/project.pbxproj

```diff
@@ -381,7 +381,7 @@
 			repositoryURL = "https://github.com/whereby/ios-sdk.git";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 0.5.0;
```

Comment:
Strictly speaking it's not really needed to change the version in the project unless you upgrade to the new major version. You can trigger the package version update from Xcode and then the new version is saved in Package.resolved file.